cardano_node_tests/tests/conftest.py

```diff
@@ -236,17 +236,18 @@ def session_autouse(change_dir: Any, close_dbconn: Any, testenv_setup_teardown:
 
 
 @pytest.fixture(scope="module")
-def testfile_temp_dir(tmp_path_factory: TempdirFactory, this_testfile: str) -> Path:
+def testfile_temp_dir(tmp_path_factory: TempdirFactory) -> Path:
     """Return a temporary dir for storing test artifacts.
 
     The dir is specific to a single test file.
     """
-    p = (
-        temptools.get_pytest_worker_tmp(tmp_path_factory)
-        .joinpath(helpers.get_id_for_mktemp(this_testfile))
-        .resolve()
-    )
+    # get the current test file
+    file_path = (os.getenv("PYTEST_CURRENT_TEST") or "").split("::")[0].split("/")[-1]
```

Comment:
Current behavior was to replace dots with underscores, so directory is not mistaken for regular file. So e.g. temp dir for `test_cli.py` becomes `test_cli_py`. Please retain this behavior.

Please use a default directory name (e.g. "unknown") instead of empty string. A temp directory should be created even if there is a corner case where the `PYTEST_CURRENT_TEST` is not set.